util/stmtsummary/statement_summary.go

```diff
@@ -361,7 +361,7 @@ func (ssMap *stmtSummaryByDigestMap) GetMoreThanOnceSelect() ([]string, []string
 		func() {
 			ssbd.Lock()
 			defer ssbd.Unlock()
-			if ssbd.initialized && ssbd.stmtType == "Select" {
+			if ssbd.initialized && (ssbd.stmtType == "Select" || ssbd.stmtType == "Delete" || ssbd.stmtType == "Update") {
```

Comment:
A question that not about the th pr's main content:
Shoudl we check `Setopr` here?

Reply:
4.0 does not support `SetOpr` now, we can do it in another PR.